easybuild/easyconfigs/g/GSL/GSL-2.1-CrayGNU-2015.11.eb

```diff
@@ -0,0 +1,20 @@
+# Built with EasyBuild version 2.8.1 on 2016-06-16_13-37-02
+easyblock = 'ConfigureMake'
+
+name = 'GSL'
+version = '2.1'
+
+homepage = 'http://www.gnu.org/software/gsl/'
+description = """The GNU Scientific Library (GSL) is a numerical library for C and C++ programmers.
+ The library provides a wide range of mathematical routines such as random number generators, special functions
+ and least-squares fitting."""
+
+toolchain = {'name': 'CrayGNU', 'version': '2015.11'}
+toolchainopts = {'opt': True, 'optarch': True, 'unroll': True, 'pic': True}
```

Comment:
Can you check on the build logs if this toolchain option 'unroll' is translated to the correct flags (CFLAGS variable for example)?

I'm not sure if this is implemented on CrayGNU.. If it is not working you will probably see `-True` added to the flags and in that case I think it is better to remove `'unroll': True` from the easyconfig file.